src/Raven.Studio/wwwroot/index.html

```diff
@@ -60,6 +60,10 @@
         <script type="text/javascript" src="lib/knockout-delegatedEvents/build/knockout-delegatedEvents.js"></script>
         <script type="text/javascript" src="lib/knockout-postbox/build/knockout-postbox.js"></script>
         <script type="text/javascript" src="lib/moment/moment.js"></script>
+        <script type="text/javascript" src="lib/cronstrue/dist/cronstrue.min.js"></script>
+        <script>
+            var cRonstrue = window.cronstrue;
```

Comment:
ok looks like this library can be loaded dynamically so let's use require to loading and then we can't remove it from index.html.